src/20_Components/amp-gfycat.html

```diff
@@ -0,0 +1,46 @@
+<!-- 
+  #### Introduction
+
+  Embed gfycat in your AMP HTML files.
```

Comment:
Also add a link to the AMP docs. Maybe rephrase to:

Use the amp-gfycat component to embed animated GIFs from gfycat.